### PR TITLE
Errata and boilerplate with reverted submodule

### PR DESCRIPTION
**Pull request by gkellogg**
This is the same as #7, but reverts adding the submodule using saved copies from https://github.com/w3c/rdf-common. In the GH Action, the submodule is added and the contents compared, which will fail if the local copy is not the same as that in the rdf-common repo.

[GitHack-based preview](https://raw.githack.com/w3c/rdf-concepts/revert-submodule/spec/index.html).

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Feb 1, 2023, 10:11 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Frdf-concepts%2F77a79f0705b3fadb7637c1a4a8e4d4b938d6b8db%2Fspec%2Findex.html%3FisPreview%3Dtrue)

```

😭  Sorry, there was an error generating the HTML. Please report this issue!
Specification: http://labs.w3.org/spec-generator/uploads/k0AlWq/spec/index.html?isPreview=true%3FisPreview%3Dtrue&publishDate=2023-02-01
ReSpec version: 32.7.0
File a bug: https://github.com/w3c/respec/
Error: Error: Evaluation failed: Timeout: document.respec.ready didn't resolve in 28238ms.
    at ExecutionContext._ExecutionContext_evaluate (/u/spec-generator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:229:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (/u/spec-generator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:107:16)
    at async generateHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:221:12)
    at async toHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:92:18)
    at async Object.generate [as respec] (file:///u/spec-generator/generators/respec.js:15:44)
    at async file:///u/spec-generator/server.js:252:48

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/rdf-concepts%2310.)._
</details>

### CODE REVIEW

**Comment by gkellogg** [comment]
(Of course, this fails due to the unavailability of respec.org, which is used in the ReSpec API.

**Comment by gkellogg** [comment]
@afs @pchampin please consider changes to [auto-publish.yml](https://github.com/w3c/rdf-concepts/pull/10/files#diff-b74e3f4a102d14bd23f5656b68b88f48055b92881a27c673c018c3ba8403ab15) for alternative to using submodule. Still uses it for comparison in GH Action.

**Review by afs — APPROVED** [review]

**Comment by gkellogg** [comment]
W3C Infrastructure having some problems, so Spec Generator is down right now, preventing investigation of what led to the PR Preview failure.

**Comment by gkellogg** [comment]
@pchampin @TallTed I'd like to merge this and defer the issues raised in https://github.com/w3c/rdf-concepts/pull/7#discussion_r1091284122. @TallTed perhaps you can create an issue that captures your concerns and we can tag in the text for future consideration, but it is a meta-issue, as it affects numerous other specs.

Note that the changes for dealing with spec/common will need to be replicated in other specs, which is a bit painful, but needs to be done.

File: spec/index.html [hunk 1]
```diff
@@ -1243,42 +1254,217 @@ <h2>Generalized RDF Triples, Graphs, and Datasets</h2>
...
-    Scott Bauer, Dan Brickley, Gavin Carothers, Pierre-Antoine Champin,
-    Olivier Corby, Richard Cyganiak, Souripriya Das, Ian Davis, Lee Feigenbaum,
-    Fabien Gandon, Charles Greer, Alex Hall, Steve Harris, Sandro Hawke,
-    Pat Hayes, Ivan Herman, Nicholas Humfrey, Kingsley Idehen, Gregg Kellogg,
-    Markus Lanthaler, Arnaud Le Hors, Peter F. Patel-Schneider,
-    Eric Prud'hommeaux, Yves Raimond, Nathan Rixham, Guus Schreiber,
-    Andy Seaborne, Manu Sporny, Thomas Steiner, Ted Thibodeau, Mischa Tuffield,
-    William Waites, Jan Wielemaker, David Wood, Zhe Wu, and Antoine Zimmermann.</p>
+  <section class="informative">
+    <h3>Acknowlegements for RDF 1.0</h3>
```
Comment (afs) [inline spec/index.html:1347]:
```suggestion
    <h3>Acknowledgements for RDF 1.0</h3>
```

File: spec/index.html [hunk 2]
```diff
@@ -1243,42 +1254,217 @@ <h2>Generalized RDF Triples, Graphs, and Datasets</h2>
...
+      Paul Resnick (University of Michigan), Bill Roberts (KnowledgeCite),
+      i Tsuyoshi Sakata (Digital Vision Laboratories), Bob Schloss (IBM),
+      Leon Shklar (Pencom Web Works), David Singer (IBM), Wei (William) Song (SISU),
+      Neel Sundaresan (IBM), Ralph Swick (W3C), Naohiko Uramoto (IBM),
+      Charles Wicksteed (Reuters Ltd.), Misha Wolf (Reuters Ltd.) and
+      Lauren Wood (SoftQuad).</p>
+  </section>
+
+  <section class="informative">
+    <h3>Acknowlegements for RDF 1.1</h3>
```
Comment (afs) [inline spec/index.html:1408]:
```suggestion
    <h3>Acknowledgements for RDF 1.1</h3>
```

**Pull request merged**